buildSrc/src/main/kotlin/convention.publishing.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import com.redmadrobot.build.dsl.*
+
+plugins {
+    id("com.redmadrobot.publish-config")
+    id("com.redmadrobot.publish")
+}
+
+group = "com.redmadrobot.extensions"
+version = "1.0.0"
+
+redmadrobot {
+    publishing {
+        signArtifacts = !isRunningOnCi
+        pom {
+            setGitHubProject("RedMadRobot/textvalue")
+
+            licenses {
+                mit()
+            }
+
+            developers {
+                developer(id = "osipxd", name = "Osip Fatkullin", email = "o.fatkullin@redmadrobot.com")
+                developer(id = "sonulen", name = "Andrey Tolmachev", email = "sonulen@gmail.com")
+            }
+        }
+    }
+}
+
+publishing {
+    repositories {
+        if (isRunningOnCi) githubPackages("RedMadRobot/textvalue")
+        if (isReleaseVersion && credentialsExist("ossrh")) ossrh()
+    }
+}
+
+kotlinCompile {
+    kotlinOptions {
+        moduleName = "redmadrobot.${project.name}"
```

Comment:
This could be removed because the name of the library is not conflicting with androidx libs anymore